modin/numpy/test/test_array_axis_functions.py

```diff
@@ -122,6 +137,13 @@ def test_min():
     numpy_result = numpy_arr.min(keepdims=True)
     assert modin_result.shape == numpy_result.shape
     numpy.testing.assert_array_equal(modin_result._to_numpy(), numpy_result)
+    numpy_arr = numpy.array([1, -10000, 2, 3, 4, 5])
+    modin_arr = np.array(numpy_arr)
+    numpy_mask = numpy.array([True, False, True, True, True, True])
+    modin_mask = np.array(numpy_mask)
+    assert numpy_arr.max(where=numpy_mask, initial=5) == modin_arr.max(
```

Comment:
```suggestion
    assert numpy_arr.min(where=numpy_mask, initial=5) == modin_arr.min(
```
I don't see a point of testing .`max()` within `test_min()` function 😉

Reply:
Good catch, thank you!